components/timeline/assets/Timeline.scss

```diff
@@ -0,0 +1,60 @@
+@import "@hods/sass-base";
+
+.hods-timeline {
+  overflow: hidden;
+  position: relative;
+
+  &__item {
+    @include govuk-responsive-padding(2, "bottom");
+    padding-left: 22px;
+    position: relative;
+
+    &:hover {
+      background-color: govuk-colour("mid-grey");
+    }
+
+    &:before { /* styling for lines in margin */
+      background-color: govuk-colour("blue");
+      content: "";
+      @include govuk-responsive-padding(1, "top");
+      @include govuk-responsive-padding(3, "right");
+      left: 0;
+      position: absolute;
+      @include govuk-responsive-margin(1, "top");
```

Comment:
Now I look at it, I wonder if this would be better off as:
```css
margin-top: 0.5em;
```
If I'm right, you could probably do something similar with the padding.

Reply:
0.55em works better for the margins (very subtle, but 0.5 didn't look quite right). Have now changed this. 0.3em is better for the padding - have made this change also